multiraft/multiraft.go

```diff
@@ -357,6 +359,17 @@ func (m *MultiRaft) ChangeGroupMembership(groupID proto.RangeID, commandID strin
 	return ch
 }
 
+// CoincideRaftLeader try to make the raft leader located to the same
```

Comment:
// CoincideRaftLeader attempts to elect the current node as the Raft leader,
    // and is called after the current node becomes the range leader.